components/global/portal/io.cellery.observability.ui/src/components/tracing/search/index.js

```diff
@@ -133,9 +133,10 @@ class TraceSearch extends React.Component {
                                     <Grid container justify={"flex-start"} spacing={24}>
                                         <Grid item xs={3}>
                                             <FormControl className={classes.formControl} fullWidth={true}>
-                                                <InputLabel htmlFor="cell" shrink={true}>Cell</InputLabel>
-                                                <Select value={filter.cell} inputProps={{name: "cell", id: "cell"}}
-                                                    onChange={this.getChangeHandlerForString("cell")}>
+                                                <InputLabel htmlFor="instance" shrink={true}>Instance</InputLabel>
+                                                <Select value={filter.cell}
+                                                    inputProps={{name: "instance", id: "instance"}}
+                                                    onChange={this.getChangeHandlerForString("instance")}>
```

Comment:
This should not be changes to instance. It should match the field in the state.

Reply:
fixed